tests/workloads-spin-registry-push/workload-spin-registry-push.yaml

```diff
@@ -0,0 +1,212 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wasm-spin
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: wasm-spin
+  template:
+    metadata:
+      labels:
+        app: wasm-spin
+    spec:
+      runtimeClassName: wasmtime-spin
+      containers:
+        - name: testwasm
+          image: test-registry:5000/spin-registry-push/spin-hello-world:latest
+          imagePullPolicy: IfNotPresent
+          command: ["/"]
+          resources: # limit the resources to 128Mi of memory and 100m of CPU
```

Comment:
Is there a reason we use resource limits here but not in the other deployments?

Reply:
this is copied as is from the existing testcases.